Read Templates from JSON files (#1185)
py-pdf/fpdf2: CHANGELOG.md

## [2.8.0] - Not released yet
### Added

> * [`Templates`](https://py-pdf.github.io/fpdf2/fpdf/Templates.html) can now be also defined in JSON files.

Reviewer: ```suggestion
* [`Templates`](https://py-pdf.github.io/fpdf2/fpdf/Templates.html) can now also be feed content from JSON files.
```

Author: That doesn't sound more correct to me (and I don't even mean the grammar... 😉 ).

In my mind, the csv/json files or python dicts contain the definition of the template. Later it may get fed custom text strings.
